argilla-server/src/argilla_server/jobs/queues.py

```diff
@@ -13,12 +13,16 @@
 #  limitations under the License.
 
 import redis
+from redis.cluster import RedisCluster
 
 from rq import Queue
 
 from argilla_server.settings import settings
 
-REDIS_CONNECTION = redis.from_url(settings.redis_url)
```

Comment:
Consider using early returns to simplify the logic when assigning `REDIS_CONNECTION`.